itou/employee_record/models.py

```diff
@@ -478,6 +493,17 @@ def batch_line_number(self):
 
         return self._batch_line_number
 
+    @staticmethod
+    def siret_from_asp_source(siae):
+        """
+        Fetch SIRET number or ASP source structure ("mother" SIAE)
+        """
+        if siae.source == Siae.SOURCE_USER_CREATED:
```

Comment:
Il y a un troisième cas `siae.source == Siae.SOURCE_STAFF_CREATED`.
Dans ce cas la plupart du temps la structure n'a pas de convention donc fait péter `ERROR_NO_CONVENTION_AVAILABLE` plus haut.
Mais il peut arriver qu'elle ait une convention. Auquel cas ici il faut aussi aller repêcher le SIRET de la SIAE mère et ne surtout pas retourner le SIAE de la structure créé par le support.

Bref changer en `if siae.source in [Siae.SOURCE_USER_CREATED, Siae.SOURCE_STAFF_CREATED]:`

Reply:
Plus simple, élégant et future proof : `if siae.source != Siae.SOURCE_ASP:`
(if it's not the mother, go get the mother)